resources/install.rb

```diff
@@ -223,11 +225,20 @@ def hab_path
   end
 
   def hab_command
-    cmd = if node['kernel']['release'].to_i < 3
-            ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{hab_version} -t x86_64-linux-kernel2"]
-          else
-            ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{hab_version}"]
-          end
+    if new_resource.hab_version
+      cmd = if node['kernel']['release'].to_i < 3
+              ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{new_resource.hab_version} -t x86_64-linux-kernel2"]
+            else
+              ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{new_resource.hab_version}"]
+            end
+    end
+    unless new_resource.hab_version
```

Comment:
would an else not be appropriate here?
```suggestion
    else
```

Reply:
I tried that but, cookstyle didn't seem to like it. I will try again, I was busy with a customer when I did it initially. Likely had bad formatting

Reply:
gonna respond on following question